### PR TITLE
Remove 'trusted certs'

### PR DESCRIPTION
**Pull request by jblomer**
The "trusted_certs" were an early attempt to use X.509 certificates for the top-level signature (whitelist signature). This was never fully implemented nor used and can be removed.

### CODE REVIEW

File: cvmfs/server_tool.cc [hunk 1]
```diff
@@ -42,7 +42,7 @@ bool ServerTool::InitDownloadManager(const bool follow_redirects,
 }
 
 bool ServerTool::InitVerifyingSignatureManager(
-    const std::string &pubkey_path, const std::string &trusted_certs) {
+    const std::string &pubkey_path, const std::string &/*trusted_certs*/) {
```
Comment (vvolkl) [inline cvmfs/server_tool.cc]: Why not remove the second argument alltogether? There's no interface to fullfill here, right?

Comment (jblomer) [reply cvmfs/server_tool.cc]: That's right, we'll actually do it with #3538 but we can do it here, too.

**Comment by vvolkl** [comment]
@cernvm-bot cloudtest

**Comment by jblomer** [comment]
@cernvm-bot cloudtest

**Comment by vvolkl** [comment]
@cernvm-bot cloudtest

**Comment by vvolkl** [comment]
@cernvm-bot cloudtest

**Pull request merged**